build/__variables.sh

```diff
@@ -4,7 +4,7 @@ set -e
 declare -r __REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
 
 # VSTS environment variables
-declare -r BUILD_NUMBER="$BUILD_BUILDNUMBER"
+declare -r BUILD_NUMBER="${BUILD_BUILDNUMBER:-latest}"
```

Comment:
I believe this change could have impact in other places of code(also `latest` thing feels docker specific), I would suggest to make a new variable(tag?) in `build-buildimage-bases.sh` which falls back to `latest` if there is no build number.

Reply:
Fair enough